specification/network/resource-manager/Microsoft.Network/stable/2020-06-01/expressRouteProviderPort.json

```diff
@@ -0,0 +1,170 @@
+            "schema": {
+              "$ref": "#/definitions/ExpressRouteProviderPortListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./network.json#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "ExpressRouteProviderPortList": {
+            "$ref": "./examples/expressRouteProviderPortList.json"
+          },
+          "ExpressRouteProviderPortListByLocation": {
+            "$ref": "./examples/expressRouteProviderPortListByLocation.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ExpressRouteProviderPortProperties": {
+      "properties": {
+        "portPairDescriptor": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The name of the port pair."
+        },
+        "primaryAzurePort": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The name of the primary port."
+        },
+        "secondaryAzurePort": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The name of the secondary port."
+        },
+        "peeringLocation": {
+          "type": "string",
+          "description": "The peering location of the port pair."
+        },
+        "overprovisionFactor": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Overprovisioning factor for the port pair."
+        },
+        "portBandwidthInMbps": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Bandwidth of the port in Mbps"
+        },
+        "usedBandwidthInMbps": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Used Bandwidth of the port in Mbps"
+        },
+        "remainingBandwidthInMbps": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Remaining Bandwidth of the port in Mbps"
+        }
+      },
+      "description": "Properties of ExpressRouteProviderPort."
+    },
+    "ExpressRouteProviderPort": {
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/ExpressRouteProviderPortProperties",
+          "description": "Properties of the express route Service Provider Port."
+        },
+        "etag": {
+          "readOnly": true,
+          "type": "string",
+          "description": "A unique read-only string that changes whenever the resource is updated."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "./network.json#/definitions/Resource"
```

Comment:
these are tracked resources with tags and location?  They must be in a resource group if that is true.  They also cannot be created behind the scenes, only created through ARM APIs.  Did you intend them to be proxy resources instead?

Reply:
These are (proxy) resources that are created internally and customers should be able to view them.  should we be removing etag for them?

Reply:
You should not be inheriting from a "Resource" definition that supports "location" and "Tags" if this resource is proxy only.  Proxy resources do not support those properties